packages/devtools_app/lib/src/charts/flame_chart.dart

```diff
@@ -118,19 +118,19 @@ abstract class FlameChartState<T extends FlameChart,
 
   bool _altKeyPressed = false;
 
-  double mouseHoverX;
+  double? mouseHoverX;
 
-  FixedExtentDelegate verticalExtentDelegate;
+  late final FixedExtentDelegate verticalExtentDelegate;
 
-  LinkedScrollControllerGroup verticalControllerGroup;
+  late final LinkedScrollControllerGroup verticalControllerGroup;
 
-  LinkedScrollControllerGroup horizontalControllerGroup;
+  late final LinkedScrollControllerGroup horizontalControllerGroup;
 
-  ScrollController _flameChartScrollController;
+  ScrollController? _flameChartScrollController;
```

Comment:
this can be late final. it is initialized in initState

Reply:
done